### PR TITLE
Homepage

### PR DESCRIPTION
**Pull request by muskanbagrecha**
Added UI for homepage.

Files to view: [files](https://github.com/muskanbagrecha/quiz-react-app/pull/1/files?file-filters%5B%5D=.css&file-filters%5B%5D=.js&file-filters%5B%5D=.jsx&show-viewed-files=true)

PS: Routing will be completed in the next PR and images are not final.

### CODE REVIEW

File: src/components/UI/navigation/Navigation.jsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import { NavLink } from "react-router-dom";
+import "./Navigation.css";
+import logo from "../../../assets/logo.png";
+export const Navigation = () => {
+  return (
+    <nav className="navigation">
+      <a href="index.html">
```
Comment (zoef-yt) [inline src/components/UI/navigation/Navigation.jsx:7]: As this page doesn't exist clicking on the logo will take you to 404 page.

**Review by zoef-yt — APPROVED** [review]
The code is really good. Left some suggestions you can look into and get back to me with your opinion. The UI looks amazing. Good work muskan. You are good to merge.

File: src/components/UI/navigation/Navigation.css [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+    padding: 0 1rem;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+}
+
+.searchbar {
+    display: flex;
+    align-items: center;
+    justify-content: center;
```
Comment (SinchanaMS) [inline src/components/UI/navigation/Navigation.css:12]: You could put this in util as it's been used multiple times.

**Review by SinchanaMS — APPROVED** [review]
The UI looks great! The code is clean as well. Great work, Muskan!!

File: src/hooks/useFetch.jsx [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+export const useFetch = () => {
+  const [data, setData] = useState(null);
+  const [loading, setLoading] = useState(false);
+  const [error, setError] = useState(null);
+  const serverCall = async (params) => {
+    try {
+      setLoading(true);
+      setError(null);
+      const res = await axios.request(params);
+      setData(res?.data);
```
Comment (ritikaawasthy) [inline src/hooks/useFetch.jsx:13]: why is there a question mark here?

File: src/pages/homepage/components/category/CategorySection.css [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+    justify-content: center;
+}
+
+.card-category .card__img{
+    background-color: transparent;
+}
+
+.card-category .card__img img{
+    height: 14rem;
+    background: white;
```
[the remark this answers is not in the document]
Comment (ritikaawasthy) [reply src/pages/homepage/components/category/CategorySection.css:60]: Agreed, it would be easier to change colors in future if you need to

**Review by ritikaawasthy — COMMENTED** [review]
Your UI looks really clean and CSS it very well written with consistent naming. Do check out the comments i made

File: src/components/UI/footer/Footer.css [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+}
+
+.footer__list .fa:hover {
+    color: var(--color-primary);
+    transition: color 0.3s ease-in;
+}
+
+.copyright {
+    font-size: var(--fsize-200);
+    color: rgba(var(--color-complementary-values), 0.8);
```
[the remark this answers is not in the document]
Comment (muskanbagrecha) [reply src/components/UI/footer/Footer.css:38]: Thank you for the detailed explanation.

File: src/hooks/useFetch.jsx (hunk 3 above)
Comment (muskanbagrecha) [reply src/hooks/useFetch.jsx:13]: optional chaining

**Pull request merged**